src/main/java/com/sap/sgs/phosphor/fosstars/model/value/PackageManagers.java

```diff
@@ -144,7 +144,8 @@ public int hashCode() {
 
   @Override
   public String toString() {
-    return packageManagers.stream().map(Enum::toString).collect(Collectors.joining(", "));
+    return packageManagers.stream()
+        .sorted().map(Enum::toString).collect(Collectors.joining(", "));
```

Comment:
There seems to be no difference in the line.

Reply:
I added `sorted()` call.

Reply:
But looks like this call is not necessary anymore since the class uses `TreeSet`. I'll remove it.

Reply:
👍